src/styles/header/_nav.scss

```diff
@@ -0,0 +1,71 @@
+@import "./variables";
+
+.header-content {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.logo-container {
+  margin-top: 10px;
+}
+
+.nav {
+
+  &__link {
+    text-decoration: none;
+    padding: 23px 0 22px;
+    color: $firstcolor;
+    font-weight: 500;
+    transition: 300ms;
+
+    &:hover {
+      color: $forthcolor;
+
+    }
+  }
+
+  &__list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    text-transform: uppercase;
+    padding-left: 0;
+    margin-top: 22.5px;
+  }
+
+  &__item {
+    margin-right: 20px;
+    color: $fifthcolor;
+    position: relative;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+  }
+
+}
+
+[data-qa="nav-hover"] {
+  color: $forthcolor;
+}
```

Comment:
I think it doesn't need to be blue all the time. The tests seem not to like this either.